app/components/autocomplete_selector/autocomplete_selector.js

```diff
@@ -167,18 +171,28 @@ export default class AutocompleteSelector extends PureComponent {
         let helpTextContent;
         if (helpText) {
             helpTextContent = (
-                <Text style={style.helpText}>
-                    {helpText}
-                </Text>
+                <View style={style.helpTextContainer} >
+                    <Markdown
+                        baseTextStyle={style.helpText}
+                        textStyles={textStyles}
+                        blockStyles={blockStyles}
+                        value={helpText}
```

Comment:
same as above

Reply:
Highlights and hashtags I think we should disable. Channel and User mentions... I see value, but I am not 100% sure. Gallery the same. I see value the moment we have the image metadata passed through.

Reply:
maybe give it a try and see what is the UX when you tap on them